test/Controller/ArticleControllerTest.php

```diff
@@ -586,9 +585,11 @@ public function it_displays_metrics()
             [
                 'Cited 1,234',
                 'Views 5,678',
-                'Comments 0',
+                'Annotations 0 Open annotations (there are currently 0 annotations on this page).',
```

Comment:
a bit underconstrained (multiple values could end up producing this) but unlikely to be a false positive

Reply:
I didn't worry too much about this as I think ELPP-3404 will remove it from the raw HTML anyway.